src/jit/lsra.h

```diff
@@ -380,8 +380,10 @@ struct LsraBlockInfo
     // 0 for fgFirstBB.
     unsigned int         predBBNum;
     BasicBlock::weight_t weight;
-    bool                 hasCriticalInEdge;
-    bool                 hasCriticalOutEdge;
+    bool                 hasCriticalInEdge : 8;
```

Comment:
I haven't measured; I'll do that.

Reply:
@sandreenko - thanks for reviewing this again. I was surprised that measurements showed that there was a small but measurable (~ .17%) throughput impact (crossgen of SPC.dll) for this overall change, but backing this out this particular change had zero measurable impact. As it happens, the refactoring of the `regsToFree` was the culprit, so I've taken out that change. I guess it shouldn't be too terribly surprising. Even though it was mostly a refactoring (and persisted even when I removed everything but the abstraction into a struct instead of two separate locals), it is on the critical path for register allocation. I'll definitely have to do some tuning on the actual EH write-thru changes, since they will impact that same code path.